python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -272,152 +232,72 @@ def __init__(self, *args, **kwargs):
     @_dask_cudf_nvtx_annotate
     @_check_groupby_supported
     def count(self, split_every=None, split_out=1):
-        return groupby_agg(
-            self.obj,
-            self.by,
-            {self._slice: "count"},
-            split_every=split_every,
-            split_out=split_out,
-            sep=self.sep,
-            sort=self.sort,
-            as_index=self.as_index,
-            **self.dropna,
-        )[self._slice]
+        return _make_groupby_agg_call(self, self.by, {self._slice: "count"})[
```

Comment:
Seems like `split_every` and `split_out` aren't being passed through for any of the `Series` calls.